.github/workflows/build.yml

```diff
@@ -34,13 +34,13 @@ jobs:
         - os: ubuntu-latest
           package_extension: 'tar.xz'
           package_suffix: 'linux64'
-          ff7tk_pack_suffix: "Linux_x86_64.tar.gz"
+          ff7tk_pack_suffix: "linux_x86_64.tar.gz"
           ff7tk_install_command_extract: "tar -xf"
           ff7tk_install_command_dest: "--one-top-level="
         - os: macos-latest
           package_extension: 'dmg'
-          package_suffix: 'macos64'
-          ff7tk_pack_suffix: "macos_x86_64.zip"
+          package_suffix: 'macos'
```

Comment:
Now I see this change, I wonder if it is OK to do it: Apple changed the main architecture of its processors 4 times now (powerpc, x86, x86_64 and currently a homemade ARM arch), if one plate-form needs to indicate the arch somewhere, that's will be Mac

Reply:
the mac build has both x86_64 and arm compatibilty. there is no compatibilty for PPC or  x86(32-bit) in any verison of mac os this build will run on.